core/core/sustainability_labels/bootstrap_database.py

```diff
@@ -10,7 +10,7 @@
 def _get_localized_certificate_attribute(
     certificate_information: Dict[str, Dict[str, Any]],
     attribute: str,
-    language_order: List[str] = ["de", "en", "fr"],
+    language_order: List[str] = ["de", "en", "fr"],  # TODO: replace with enum?
```

Comment:
These have to be lower cased. Already had it based on the enums but it gets ugly. Further, we do not have something for "en".
I guess for now it's fine, wdyt?

Reply:
Yes, I think it's fine for now 👍